docs/source/output_sanitize.cfg

```diff
@@ -39,13 +39,35 @@ regex: 0x[0-9a-f]+>
 replace: 0xOBJECT_HASH>
 
 [regex11]
-regex: pywps_process_[a-z0-9_]{8}
-replace: PYWPS_PROCESS_CODE
-
-[regex12]
 regex: \/\/www\.w3\.org\/TR\/xmlschema-2\/#[a-z]+
 replace: W3_DATA_TYPE
 
-[regex13]
-regex: \/home/[a-zA-Z0-9_]+\/[.]?[a-zA-Z0-9_]+\/
-replace: /home/USERNAME/ANACONDA_LOCATION/
+[geo.weather.gc.ca-WCS-ver]
```

Comment:
@Zeitsperre you forgot to delete this local file `docs/source/output_sanitize.cfg` in this commit.  Have to also add it to `.gitignore` so it does not come back into future commits.

Reply:
This is being dealt with in #224 . Thanks!